minecraft/1.7/src/main/java/nova/core/wrapper/mc/forge/v17/wrapper/entity/backward/BWEntity.java

```diff
@@ -64,6 +66,10 @@ public void damage(double amount, DamageType type) {
 				living.faceDisplacement = () -> Vector3D.PLUS_J.scalarMultiply(entity.getEyeHeight());
 			}
 		}
+		if (entity != null) {
```

Comment:
This line wouldn’t have prevented a NPE anyway, as that would have been thrown already